DESCRIPTION

```diff
@@ -16,19 +16,15 @@ Authors@R: c(
            comment = c(ORCID = "0000-0002-6131-8165"))
   )
 Maintainer: Naeem Khoshnevis <nkhoshnevis@g.harvard.edu>
-Description: Provides an interpretable identification of subgroups with
-    heterogeneous causal effect. The heterogeneous subgroups are
-    discovered through ensemble learning of causal rules. Causal rules are
-    highly interpretable if-then statement that recursively partition the
-    features space into heterogeneous subgroups. A small number of
-    significant causal rules are selected through Stability Selection to
-    control for family-wise error rate in the finite sample setting. It
-    proposes various estimation methods for the conditional causal effects
-    for each discovered causal rule.  It is highly flexible and multiple
-    causal estimands and imputation methods are implemented.  Lee, K.,
-    Bargagli-Stoffi, F. J., & Dominici, F. (2020). Causal rule ensemble:
-    Interpretable inference of heterogeneous treatment effects.  arXiv
-    preprint <arXiv:2009.09036>.
+Description: Causal Rule Ensemble is a new method for interpretable 
```

Comment:
Please start with provides and drop Causal Rule Ensemble.

Reply:
fixed